proto/context/src/main/proto/spine/validation/events.proto

```diff
@@ -71,11 +72,11 @@ message RequiredFieldDiscovered {
 
     protodata.FieldRef id = 1;
 
-    // The error message template.
-    string error_message = 2;
-
     // The field in which the option was discovered.
-    protodata.Field subject = 3;
+    protodata.Field subject = 2;
```

Comment:
I've re-ordered fields, so that the most common `id, subject, error_message` go first, and the option-specific fields go last.